### PR TITLE
Few improvements and bug fix.

### PR DESCRIPTION
**Pull request by xuhao1**
Fix running bug: https://github.com/taichi-dev/taichi_three/issues/44
Better orbit system: based on manifold (quaternion differentiation), no gimbal lock now.
Enable different resolution on width and height.
Add transformations.py

### CODE REVIEW

File: tina/util/transformations.py [hunk 1]
```diff
@@ -0,0 +1,1705 @@
+# -*- coding: utf-8 -*-
```
Comment (k-ye) [inline tina/util/transformations.py:1]: Why not just install https://pypi.org/project/transformations/ ?

**Review by k-ye — COMMENTED** [review]
Thanks! I have one question around the `transformations` module.

File: tina/util/transformations.py (hunk 1 above)
Comment (xuhao1) [reply tina/util/transformations.py]: Sure. I update the code!

**Comment by xuhao1** [comment]
Add install require: transformations. Remove the transformations.py
Support ortho on new orbit control.

**Comment by k-ye** [comment]
Thanks!

**Pull request merged**